src/main/java/uk/gov/companieshouse/api/accounts/service/impl/RptTransactionServiceImpl.java

```diff
@@ -155,8 +161,6 @@
 
                 repository.deleteById(rptTransactionId);
 
```

Comment:
When we delete a transaction we will need to remove it from the parent resource, similar to LTD loans delete functionality